force-app/main/default/triggers/LeadTrigger.trigger

```diff
@@ -15,7 +15,7 @@
  * - It's essential to test the trigger thoroughly after making any changes to ensure its correct functionality.
  * - Debugging skills will be tested, so students should look out for discrepancies between the expected and actual behavior.
  */
-trigger LeadTrigger on Lead(before insert) {
+trigger LeadTrigger on Lead(before insert, before update, after insert, after update) {
```

Comment:
It's a little advanced but try to implement the trigger handler framework
Source: https://github.com/kevinohara80/sfdc-trigger-framework/tree/master